comment_spell_check/utils/__init__.py

```diff
@@ -0,0 +1 @@
+# he/him/his
```

Comment:
This should just be an empty file, no comment needed.

Reply:
Too woke?

Reply:
Done